contrib/IECoreUSD/test/IECoreUSD/SceneCacheFileFormatTest.py

```diff
@@ -1059,6 +1059,7 @@ def testReferenceUnsupportedExtension( self ):
 	def testLinksTimeOffset( self ):
 		fps = 24.0
 		fileName = os.path.join( self.temporaryDirectory(), "testUSDTimeOffsetLinked.scc" )
+		fileName = fileName[:1].lower() + fileName[1:] if os.name == "nt" else fileName
```

Comment:
I think this is what `os.path.normcase()` is for - at least that's what I've used elsewhere in the USD tests. Rather than changing the path here, I think it might be better to use `normcase()` at the point we do the assertion on L1087 (to show that case on input doesn't matter, but that you can't expect case to be maintained on output).

Reply:
Very nice, I didn't know about `normcase` and `normpath`, those will be handy. Fixed in c8c04b33c97e8f8824da450e34730de4cfa55d34